index.html

```diff
@@ -0,0 +1,147 @@
+                              <li><a class="listitems1" href="#">Program</a></li>
+                              <li><a class="listitems1" href="#">Join</a></li>
+                              <li><a class="listitems1" href="#">Sponsor</a></li>
+                              <li><a class="listitems1" href="#">News</a></li><li>
+                              <li><a class="listitems2" href="#">Campaign</a></li>
+                          </ul>
+                      </div>
+          
+                  </section>
+                <img src="img/hamburger.png" alt="hamburger" class="hamburgerimage2" id="pop">
+            </nav>
+    </header>
+    <Section class="sectiontitle"> 
+        <h1 class="titlei">  "Hi, Explore your world"</h1>
+    <div class="desc1"> 
+        <h1 class="titleheading">Web Development global training summit 2022</h1>
+        <P class="titledetail"> Become an expert in web development that the world top tech company seeks for, 
+        through our organized trianing packed with first-class experience and hands on 
+        real projects, strictly surpvised by our programmers. </P>
+    </div> 
+    <div><H2 class="date">2022.05.02(MON) ~ 05 (THUR)</H2>
+    <P class=" ivenue">@ Era Auditorium of Nigeria, Technology Street Enugu</P></div>
+    </Section>
+    <section class="programsection">
+        <div><p class="main">Main Program</p></div>
+        <div class="speaker1m-hr"> </div>
+        <div class="main-program">
+            <div class="program">
+                <img  class= "imglecture" src="img/program_icon_01.png" alt="lecture image">
+                <h3>Lectures</h3>
+                <p class="desc3"> There will lectures on different programming languages both for the front end and back end 
+                    development. 
+                </p>
+            </div>
+            <div class="program"> 
+                <img src="img/program_icon_02.png" alt="Exhibition image">
+                <h3> Exhibition </h3>
+            <P class="desc3i"> The practical exhibition of the topic and it applicaion and usage is will be shown</P>
+            </div>
+            <div class="program">
+                <img src="img/program_icon_03.png" alt="Forum image">
+                <h3>Forum</h3> 
+            <p class="desc3e"> Students will be allowed to interact with the programmers and ask
+                questions. 
+            </p>
+        </div>
+        <div class="program"> 
+            <img src="img/program_icon_04.png" alt="Workshop image">
+            <h3> Workshop</h3>
+            <p class="desc3i">Students will be given some projects to work on which will supervised by programmers</p>
+        </div>
+        <div class="program">
+            <img src="img/program_icon_05.png" alt="Milestone image">
+            <h3>Milestone</h3>
+            <p class="desc3i">The project for each student to test knowledge acquired</p>
+        </div>
```

Comment:
![image](https://user-images.githubusercontent.com/60612329/164739766-df3af40f-64c3-4d54-be37-d6f95a2b5f16.png)
as seen above the following should be addressed 

- the program section should be centralized 
- the text ``see more program `` should be centralized to improve theUX of the webpage